packages/mobile/src/harmony-native/foundations/theme/ThemeProvider.tsx

```diff
@@ -19,6 +13,8 @@ type ThemeProviderProps = {
 export const ThemeProvider = (props: ThemeProviderProps) => {
   const { children, theme } = props
   return (
-    <ThemeContext.Provider value={{ theme }}>{children}</ThemeContext.Provider>
+    <EmotionThemeProvider theme={nativeHarmonyThemes[theme]}>
+      {children}
+    </EmotionThemeProvider>
   )
 }
```

Comment:
nit: This component isn't really doing much other than add another layer tbh. Personally I would prefer we just move the theme selection logic to `src/app/ThemeProvider.tsx` (where most of it is anyways) and this file just becomes a re-export of the emotion one. 
```tsx
export { ThemeProvider } from "@emotion/native"
```

Reply:
On the other hand, if you do decide to keep this component pretty much as-is, could we rename the `theme` prop to `themeName` for clarity? When I see `theme` as a variable I assume its the entire theme object

Reply:
yeah agreed it should be theme name. i kinda like it separate cause the other theme provider is manging a bunch of legacy theme stuff and i ilke the separation. but down to figure out the cleanest approach for this long term. will change themeName though for sure.